File: 8-geometria/1875.php
<?php
function spore($ISSUER,$RECEIVER):int{
	if($ISSUER==ord('R')&&$RECEIVER==ord('G'))
		return 2;
	else if($ISSUER==ord('R'))
		return 1;

	if($ISSUER==ord('G')&&$RECEIVER==ord('B'))
		return 2;
	else if($ISSUER==ord('G'))
		return 1;

	if($ISSUER==ord('B')&&$RECEIVER==ord('R'))
		return 2;
	else
		return 1;
}


$REP=(int)readline();
for($I=0;$I<$REP;++$I){
	$GOALS=(int)readline();
	$ARR=array(0,0,0);
	for($J=0;$J<$GOALS;++$J){
		$STR=readline();
		$TOKEN=strtok($STR," ");

		$STR_1_CHR=ord($TOKEN[0]);
		$TOKEN=strtok(" ");
		$STR_2_CHR=ord($TOKEN[0]);

		if($STR_1_CHR==ord('B'))
			$ARR[0]+=spore($STR_1_CHR,$STR_2_CHR);
		if($STR_1_CHR==ord('G'))
			$ARR[1]+=spore($STR_1_CHR,$STR_2_CHR);
		if($STR_1_CHR==ord('R'))
			$ARR[2]+=spore($STR_1_CHR,$STR_2_CHR);
	}
	if($ARR[0]>$ARR[1]&&$ARR[0]>$ARR[2])
		echo "blue\n";
	else if($ARR[1]>$ARR[0]&&$ARR[1]>$ARR[2])
		echo "green\n";
	else if($ARR[2]>$ARR[0]&&$ARR[2]>$ARR[1])
		echo "red\n";
	else if($ARR[0]==$ARR[1]&&$ARR[1]==$ARR[2])
		echo "trempate\n";
	else if($ARR[0]==$ARR[1]||$ARR[0]==$ARR[2]||$ARR[1]==$ARR[2])
		echo "empate\n";
}
?>

File: 8-geometria/1875.cs
using System;

class Thousand_Eight_Hundred_Seventy_five{
	static int spore(int issuer,int receiver){
		if(issuer==(int)'R'&&receiver==(int)'G')
			return 2;
		else if(issuer==(int)'R')
			return 1;

		if(issuer==(int)'G'&&receiver==(int)'B')
			return 2;
		else if(issuer==(int)'G')
			return 1;

			if(issuer==(int)'B'&&receiver==(int)'R')
			return 2;
		else
			return 1;
	}
	static void Main(){
		int rep=Convert.ToInt32(Console.ReadLine());
		for(int i=0;i<rep;++i){
			int goals=Convert.ToInt32(Console.ReadLine());
			int []array={0,0,0};
			for(int j=0;j<goals;++j){
				String big_string=Console.ReadLine();
				String[]array_str=big_string.Split(new string[]{" "},2,StringSplitOptions.RemoveEmptyEntries);

				int str_1_chr=(int)array_str[0][0],str_2_chr=(int)array_str[1][0];
				if(str_1_chr==(int)'R')
					array[0]+=spore(str_1_chr,str_2_chr);
				if(str_1_chr==(int)'G')
					array[1]+=spore(str_1_chr,str_2_chr);
				if(str_1_chr==(int)'B')
					array[2]+=spore(str_1_chr,str_2_chr);
			}
			if(array[0]>array[1]&&array[0]>array[2])
				Console.WriteLine("red");
			else if(array[1]>array[0]&&array[1]>array[2])
				Console.WriteLine("green");
			else if(array[2]>array[0]&&array[2]>array[1])
				Console.WriteLine("blue");
			else if(array[0]==array[1]&&array[0]==array[2])
				Console.WriteLine("trempate");
			else
				Console.WriteLine("empate");
		}
	}
}
